components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,86 @@
+import { Button, Drawer, List, ListItem, Typography } from '@mui/material'
+import React, { Fragment, useState } from 'react'
+import HeaderTab from './HeaderTab'
+import { sidebar_data } from './sidebar_data'
+import LogoutIcon from '@mui/icons-material/Logout'
+
+// Rename tabs
+// Hover
+// Select tabs ui feedback
+
+var drawerWidth = 320
+
+const Sidebar = () => {
+  const [open, setOpen] = useState<boolean>(true)
+  const [tabSelected, setTabSelected] = useState<string>('')
+
+  const toggleDrawer = (open: boolean) => {
+    setOpen(open)
+  }
```

Comment:
this function seems to just set the state. you can just call setOpen() directly in the components instead of toggelDrawer